deposit_contract/contracts/validator_registration.v.py

```diff
@@ -78,6 +81,10 @@ def get_deposit_count() -> bytes[8]:
 @payable
 @public
 def deposit(pubkey: bytes[48], withdrawal_credentials: bytes[32], signature: bytes[96]):
```

Comment:
Can we include the constant values in the function def? not sure if vyper lets you do it. Only downside is line length blows up

```suggestion
def deposit(pubkey: bytes[PUBKEY_LENGTH], withdrawal_credentials: bytes[WITHDRAWAL_CREDENTIALS_LENGTH], signature: bytes[SIGNATURE_LENGTH]):
```

Reply:
it seems like we in fact can, that's neat